package.json

```diff
@@ -25,6 +25,7 @@
     "@types/node": "^16.4.13",
     "@types/react": "^17.0.18",
     "@types/react-dom": "^17.0.9",
+    "@types/styled-components": "^5.1.15",
```

Comment:
I think Material UI has everything you need, so you probably won't use SC

Reply:
That would be great, and if I won't, I´ll delete the dependency, but I thought it could come in handy if I want to override mui themes.

Reply:
You should remove SC, because you can adjust/change/override MUI styles in another way, check their docs to see how :)

Reply:
Sir, yes Sir :D